tests/test_image_manager.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+import boto3
+from boto3.resources.base import ServiceResource
+from image_manager import AWSConnector
+from image_manager import ImageManager
+from moto import mock_s3
+import helper
+
+
+@mock_s3
+class TestImageManager(unittest.TestCase):
+    bucket = 'jf-test555-bucket'
+
+    def setUp(self):
+        aws_connector = AWSConnector()
+        self.s3_resource = aws_connector.get_s3_resource()
+        self.s3_resource.create_bucket(Bucket=self.bucket, CreateBucketConfiguration={
+            'LocationConstraint': 'eu-west-2'},)
+        self.image_manager = ImageManager(self.s3_resource, self.bucket)
+
+    def test_add_image(self):
+        # arrange
+        id = helper.get_id()
+        filename = "tests/test2.jpg"
+        vector_img = "vector/" + self.image_manager._set_img_name(filename, id)
+        png_img = "png/" + self.image_manager._set_img_name(filename, id)
+
+        # act
+        self.image_manager.add_image(filename, id)
+        self.image_manager.add_image(filename, id, False)
+        images = tuple(
+            img.key for img in self.image_manager.bucket.objects.all())
+
+        # assert
+        self.assertEqual(len(images), 2)
+        self.assertIn(vector_img, images)
+        self.assertIn(png_img, images)
+
+    def test_delete_image(self):
+        # arrange
+        id = helper.get_id()
+        filename = "tests/test2.jpg"
```

Comment:
Since this is used on so many places you can put this as a constant outside of the class, e.g:

```
JPG_FILENAME = "tests/test2.jpg"
```

Reply:
updated